### PR TITLE
some initial work on a NEWS file for 239

### PR DESCRIPTION
**Pull request by poettering**
As usually, let#s get merged this quickly, and submit typo fixes and additions in a follow-up PR! thanks!

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -31,10 +31,265 @@ CHANGES WITH 239 in spe:
...
+          which /var/run/ is simply a symlinked compatibility alias). This is
+          recommended, so that systemd-tmpfiles can properly detect line
+          conflicts and merge lines referencing the same files by both paths,
+          without having to access them.
+
         * systemctl disable/unmask/preset/preset-all cannot be used with
           --runtime. Previously this was allowed, but resulted in unintuitive
           behaviour that wasn't useful.
 
+        * sd-boot acquire new loader configuration settings for optionally
```
Comment (mbiebl) [inline NEWS:46]: acquired?

File: NEWS [hunk 2]
```diff
@@ -31,10 +31,265 @@ CHANGES WITH 239 in spe:
...
+        * The nss-systemd glibc NSS module will now enumerate dynamic users and
+          groups in effect. Previously, it could resolve UIDs/GIDs to user
+          names/groups and vice versa, but did not support enumeration.
+
+        * journald's Compress= configuration setting now optionally accepts a
+          byte threshold value. All journal objects larger than this threshold
+          will be compressed, smaller ones will not. Previously this threshold
+          was not configurable and set to 512.
+
+        * A new system.conf NoNewPrivileges= is now available which may be used
```
Comment (mbiebl) [inline NEWS:76]: system.conf setting? system.conf configuration option?

**Comment by keszybz** [comment]
Yeah, let's merge and adjust in later PRs. I see various typos, but doing a patch is probably less work than a review.

**Comment by keszybz** [comment]
Suggested fixes in #9268.

**Pull request merged**